bsp/imx/apalis-imx6.cfg

```diff
@@ -0,0 +1,121 @@
+CONFIG_DMA_CMA=y
+# CONFIG_INPUT_MOUSE is not set
+CONFIG_EEPROM_AT24=y
+CONFIG_EEPROM_AT25=y
+CONFIG_ATA=y
+CONFIG_SATA_AHCI_PLATFORM=y
+CONFIG_AHCI_IMX=y
+CONFIG_SMC91X=y
+CONFIG_SMC911X=y
+CONFIG_SMSC911X=y
+CONFIG_AT803X_PHY=y
+CONFIG_MICREL_PHY=y
+CONFIG_SMSC_PHY=y
+CONFIG_BRCMFMAC=m
+CONFIG_WL18XX=m
+CONFIG_WLCORE_SDIO=m
+# CONFIG_WILINK_PLATFORM_DATA is not set
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_KEYBOARD_SNVS_PWRKEY=y
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_SERIAL_DEV_CTRL_TTYPORT=y
+CONFIG_SPI_GPIO=y
+CONFIG_GPIO_PCA953X=y
+CONFIG_GPIO_74X164=y
+CONFIG_POWER_SUPPLY=y
+CONFIG_SENSORS_GPIO_FAN=y
+CONFIG_SENSORS_IIO_HWMON=y
+CONFIG_THERMAL=y
+CONFIG_THERMAL_WRITABLE_TRIPS=y
+CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_PFUZE100=y
+CONFIG_DRM=y
+CONFIG_DRM_PANEL_LVDS=y
+CONFIG_DRM_PANEL_SIMPLE=y
+CONFIG_DRM_DW_HDMI_AHB_AUDIO=m
+CONFIG_DRM_DW_HDMI_CEC=y
+CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
+CONFIG_DRM_IMX_TVE=y
+CONFIG_FRAMEBUFFER_CONSOLE=y
+CONFIG_SND_SOC=m
+CONFIG_SND_IMX_SOC=m
+CONFIG_SND_SOC_SGTL5000=m
+CONFIG_SND_SOC_EUKREA_TLV320=m
+CONFIG_SND_SOC_IMX_WM8962=m
+CONFIG_SND_SOC_IMX_ES8328=m
+CONFIG_SND_SOC_IMX_SGTL5000=m
+CONFIG_SND_SOC_IMX_SPDIF=m
+CONFIG_SND_SOC_CS42XX8_I2C=m
+CONFIG_SND_SOC_TLV320AIC3X=m
+CONFIG_SND_SOC_WM8960=m
+CONFIG_USB_EHCI_MXC=y
+CONFIG_USB_FSL_USB2=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_RTC_INTF_DEV_UIE_EMUL=y
+CONFIG_RTC_DRV_DS1307=y
+CONFIG_RTC_DRV_ISL1208=y
+CONFIG_RTC_DRV_PCF8523=y
+CONFIG_RTC_DRV_PCF8563=y
+CONFIG_RTC_DRV_M41T80=y
+CONFIG_RTC_DRV_MXC=y
+CONFIG_RTC_DRV_MXC_V2=y
+CONFIG_DMADEVICES=y
+CONFIG_FSL_EDMA=y
+CONFIG_STAGING=y
+CONFIG_STAGING_MEDIA=y
+CONFIG_COMMON_CLK_PWM=y
+CONFIG_PWM_FSL_FTM=y
+CONFIG_CRYPTO_DEV_SAHARA=y
+CONFIG_BT_HCIUART_SERDEV=y
+CONFIG_BT_HCIBTSDIO=m
+CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_PINMUX=y
+CONFIG_PINCONF=y
+CONFIG_GENERIC_PINCONF=y
+CONFIG_FB=y
+CONFIG_EXTCON_USB_GPIO=m
+CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_IR_GPIO_CIR=y
+CONFIG_LEDS_PWM=y
+CONFIG_LEDS_REGULATOR=y
+CONFIG_CAN=m
+CONFIG_I2C_GPIO=y
+CONFIG_SPI_MEM=y
+CONFIG_SPI_FSL_DSPI=y
+CONFIG_PMIC_DA9052=y
+CONFIG_MFD_DA9052_I2C=y
+CONFIG_MFD_DA9062=y
+CONFIG_MFD_RN5T618=y
+CONFIG_REGULATOR_DA9052=y
+CONFIG_REGULATOR_DA9062=y
+CONFIG_REGULATOR_MC13XXX_CORE=y
+CONFIG_REGULATOR_MC13783=y
+CONFIG_REGULATOR_MC13892=y
+CONFIG_REGULATOR_RN5T618=y
+CONFIG_VIDEO_OV5640=m
+CONFIG_USB_CHIPIDEA_PCI=y
+CONFIG_USB_CHIPIDEA_UDC=y
+CONFIG_RTC_DRV_DA9063=y
+CONFIG_RTC_DRV_MC13XXX=y
+CONFIG_ARM_ARCH_TIMER=y
+CONFIG_ARM_ARCH_TIMER_EVTSTREAM=y
+CONFIG_ARM_GLOBAL_TIMER=y
+CONFIG_CLKSRC_ARM_GLOBAL_TIMER_SCHED_CLOCK=y
+CONFIG_FIQ=y
+CONFIG_SOC_IMX6SLL=y
+# Broken DRM
+CONFIG_DRM_DW_HDMI is not set
+CONFIG_DRM_DW_HDMI_CEC is not set
+CONFIG_DRM_DW_HDMI_AHB_AUDIO is not set
+CONFIG_DRM_IMX is not set
+CONFIG_DRM_IMX_PARALLEL_DISPLAY is not set
+CONFIG_DRM_IMX_TVE is not set
+CONFIG_DRM_IMX_LDB is not set
+CONFIG_DRM_IMX_HDMI is not set
+CONFIG_DRM_ETNAVIV is not set
+CONFIG_DRM_ETNAVIV_THERMAL is not set
+CONFIG_DRM_PANEL_ORIENTATION_QUIRKS is not set
```

Comment:
Seems to be missing "# " at the beginning of every config that should be disabled. Did you check the final config to see if they were indeed disabled here?

Reply:
So it did indeed disable them, but I can add the '#' for consistency sake.

Reply:
Please, just to get it aligned with the other fragments.

Reply:
Ok, this should be done now in the latest commit.